configuration/managing-vm-kernel.md

```diff
@@ -58,10 +63,10 @@ kernel-qubes-vm-3.18.16-3.pvops.qubes.x86_64
 kernel-qubes-vm-3.18.17-4.pvops.qubes.x86_64
 ~~~
 
-If you want a more recent version, you can check `qubes-dom0-unstable` repository. As the name suggests, keep in
-mind that those packages may be less stable than the default ones.
+If you want a more recent version, you can check the `qubes-dom0-unstable` repository.
+As the name suggests, keep in mind that those packages may be less stable than the default ones.
```

Comment:
There is also `kernel-latest-qubes-vm` package - which should provide more recent (non-LTS) kernel, but much less tested.